docs/Installing.asciidoc

```diff
@@ -52,35 +49,31 @@ zypper in openQA
 
 # Fedora 23+
 dnf install openqa openqa-httpd
---------------------------------------------------------------------------------
-
-Basic configuration
--------------------
-[id="basic"]
+----
 
-Apache proxy
-~~~~~~~~~~~~
+=== Apache proxy
 
 It is required to run openQA behind an http proxy (apache, nginx, etc..). See the
 +openqa.conf.template+ config file in +/etc/apache2/vhosts.d+ (openSUSE) or
 +/etc/httpd/conf.d+ (Fedora). To make everything work correctly on openSUSE, you
 need to enable the 'headers', 'proxy', 'proxy_http' and 'proxy_wstunnel' modules
-using 'a2enmod'. This is not necessary on Fedora. For a basic setup, you can
+using the command 'a2enmod'. This is not necessary on Fedora. For a basic setup, you can
 copy +openqa.conf.template+ to +openqa.conf+ and modify the +ServerName+
 setting. This will direct all HTTP traffic to openQA.
 
 [source,sh]
 --------------------------------------------------------------------------------
 # openSUSE Only
 # You can check what modules are enabled by using 'a2enmod -l'
-a2enmod headers
-a2enmod proxy
-a2enmod proxy_http
-a2enmod proxy_wstunnel
+$ a2enmod headers
+$ a2enmod proxy
+$ a2enmod proxy_http
+$ a2enmod proxy_wstunnel
```

Comment:
doing this will make it more obvious which commands have to be entered but then copy-paste of the commands does not work anymore so I object